runtime/include/chpl-mem.h

```diff
@@ -249,14 +249,14 @@ size_t chpl_mem_localizationThreshold(void) {
 
 #else // LAUNCHER
 
-#include <stdlib.h>
+#include "chpl-mem-sys.h"
```

Comment:
Just curious -- can the check-for-malloc script ignore stuff that is `#ifdef LAUNCHER`?  If so you wouldn't need the changes here.